### PR TITLE
feat(to-lower): init toLower package

### LINKED ISSUE #6: Feature request: `toLower / toFirstLower / decapitalize`
Converts the first character of the input text to lowercase. If the second parameter is true, the method converts the rest of the subject to lower case.

```javascript
  @param {String} text
  @param {Boolean} isLowercase
```

**Example usage**

```javascript
import toLower from '@plexis/to-lower';
toLower('foo Bar');
// => foo Bar

toLower('FOO Bar');
// => fOO Bar

toLower('fOO Bar', true);
// => foo bar
```

**Aliases**

```javascript
import toLower from '@plexis/to-lower';
import {toLower, toFirstLower, decapitalize } from 'plexis';
```

### PR DESCRIPTION
**Pull request by 4doge**
Hello @vorillaz 
I've write a `toLower` package with tests, but seems like you have some issues even in your master branch with the `toPred`. Please let me know if it's my issues only and maybe some possible ways to fix that.
photo_2019-10-20 19 21 05

Fixes #6

### CODE REVIEW

**Comment by vorillaz** [comment]
Hello @4doge  and thanks very much for your contribution. Have you tried running `yarn` and `yarn bootstrap` on the root of the project ?

**Comment by 4doge** [comment]
@vorillaz sorry, my bad. I thought I run bootstrap lerna command. Looks like working now.
Thanks :)

File: packages/toLower/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# `@plexis/toLower`
+
+Lowercase the first letter of given string.
```
Comment (vorillaz) [inline packages/toLower/README.md]: There is a small typo here:
> Lowercase the first letter of the given string.

File: packages/plexis/src/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 export {default as toPred, default as toPredecessor} from '@plexis/to-pred';
 export {default as toSucc, default as toSuccessor} from '@plexis/to-succ';
 export {default as toTitle, default as titleize} from '@plexis/to-title';
+export {default as toLower} from '@plexis/to-lower';
```
Comment (vorillaz) [inline packages/plexis/src/index.js]: Can you please create the appropriate aliases as described in the corresponding issue https://github.com/plexis-js/plexis/issues/6?

File: packages/toLower/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+toLower('Foo Bar', false); // returns 'foo Bar'
+toLower('Foo Bar', true); // returns 'foo bar'
+toLower('FOO BAR', false); // returns 'fOO BAR'
+```
+
+**Aliases**
+
+```javascript
+import {toLower} from 'plexis';
```
Comment (vorillaz) [inline packages/toLower/README.md]: Same here, there are some aliases missing.

**Pull request merged**